public/javascript/getHelp.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * @file getHelp.js
+ * Handler for showing functions
+ *
+ */
+ 
+ module.exports = function setThreadColor(api, threadID, body) {
+        function sleep(milliseconds) {                  //For message timing
+                var start = new Date().getTime();
+                for (var i = 0; i < 1e7; i++) {
+                        if ((new Date().getTime() - start) > milliseconds){
+                                break;
+                        }
+                }
+        }
+        
+        var worked = 1;
+        var command = body.substring('@help'.length);
```

Comment:
I'd do `body.substring('@help '.length);` here with an extra space, and then remove the leading space in the subsequent commands. E.g. `else if(command==='meme')`.